pkgs/development/python-modules/skidl/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+, future
+, kinparse
+, enum34
+, pyspice
+, graphviz
+, pillow
+, cffi
+}:
+
+buildPythonPackage rec {
+  pname = "skidl";
+  version = "unstable-2020-09-15";
+
+  src = fetchFromGitHub {
+    owner = "xesscorp";
+    repo = "skidl";
+    rev = "551bdb92a50c0894b0802c0a89b4cb62a5b4038f";
+    sha256 = "1g65cyxpkqshgsggav2q3f76rbj5pzh7sacyhmhzvfz4zfarkcxk";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    future
+    kinparse
+    enum34
+    pyspice
+    graphviz
+    pillow
+    cffi
+  ];
+
+  # Checks require availability of the kicad symbol libraries.
+  doCheck = false;
+  pythonImportsCheck = [ "skidl" ];
+
+  meta = with stdenv.lib; {
```

preference to avoid stdenv when possible
```suggestion
  meta = with lib; {
```